pkg/pipelineascode/match.go

```diff
@@ -126,14 +126,14 @@ is that what you want? make sure you use -n when generating the secret, eg: echo
 			msg += fmt.Sprintf(" err: %s", err.Error())
 		}
 		p.eventEmitter.EmitMessage(nil, zap.InfoLevel, "RepositoryPipelineRunNotFound", msg)
-		return nil, nil, nil
+		return nil, repo, nil
 	}
 
 	// check for condition if need update the pipelinerun with regexp from the
 	// "raw" pipelinerun string
 	if msg, needUpdate := p.checkNeedUpdate(rawTemplates); needUpdate {
 		p.eventEmitter.EmitMessage(repo, zap.InfoLevel, "RepositoryNeedUpdate", msg)
-		return nil, nil, fmt.Errorf(msg)
+		return nil, repo, fmt.Errorf(msg)
```

Comment:
there was some reason which I am forgetting why i didn't returned repo in this cases 🤔

Reply:
let's return for now.. we will see 😅